toolchain/check/testdata/builtins/int/and.carbon

```diff
@@ -18,111 +20,3 @@ let arr_p: [i32; 8]* = &arr;
 fn RuntimeCall(a: i32, b: i32) -> i32 {
   return And(a, b);
 }
```

Comment:
The SemIR seams still useful, and the only real test we have for the runtime call case. Would it make sense to split the runtime call tests out to files that continue to dump SemIR and only skip it in the ones where we do compile time validation?

Reply:
I think the only thing this is testing that's specific to each builtin is that a runtime call is valid. Beside that, the handling is not specific to the particular builtin -- we get the same call SemIR regardless of the kind of builtin -- so I think it's probably good enough to test that only once rather than once per builtin.

To that end, I've gone ahead and added SemIR output coverage for runtime calls to `toolchain/check/testdata/function/builtin/call.carbon` (which previously only covered compile-time calls).

Reply:
Sure... Maybe `RuntimeCallIsValid` or something to make that a bit more obvious why there is nothing verified beyond that?